docs/2.8.1/docs/canton/usermanual/persistence.rst

```diff
@@ -622,8 +622,11 @@ Order of Backups
 
 It is important that the participant's backup is not more recent than that of
 the domain's, as that would constitute a ledger fork. Therefore, if you back up
-both participant and domain databases, always back up the participant database before the
-domain. If you are using a domain integration, then backup the sequencer node before backing
+both participant and domain databases sequentially, always back up the participant database before the
+domain. If you are able to perform a complete system backup in a single step (e.g. when using
```

Comment:
Shall we clarify here already the domain "internal" order of backups (mediator and domain manager before sequencer)?

Reply:
You mean bring it up from below?

Reply:
Not sure what that means 😬.
What I mean is the following: the sentence "_always back up the participant database before the domain_" clarifies the order between participant and domain, but isn't clear that mediator and domain manager databases (in case they are separate nodes) should be backed up before the sequencer database.

Reply:
Rephrased